homeassistant/components/overkiz/__init__.py

```diff
@@ -41,7 +41,7 @@ class HomeAssistantOverkizData:
     """Overkiz data stored in the Home Assistant data object."""
 
     coordinator: OverkizDataUpdateCoordinator
-    platforms: dict[Platform, Device | Scenario]
+    platforms: defaultdict[Platform, Device | Scenario]
```

Comment:
Shouldn't the value in the dict be a list of items?

Reply:
You are correct..